src/workloads/scale/LLTMixed.yml

```diff
@@ -241,56 +241,81 @@ Actors:
         LogEvery: 15 minutes
         Blocking: None
 
-- Name: MixedBaseline.Low
+- Name: Low.Short.Insert
   Type: CrudActor
   Threads: *LowThreads
   Phases:
     OnlyActiveInPhases:
-      Active: [5]
+      Active: [5,7]
```

Comment:
Could you add a comment to the effect of "The test is run in phase 5 (without snapshot read) and phase 7 (with snapshot read)". Assuming that is correct and it hasn't been stated some place i can't see because github PRs are "interesting".